.hashibot.hcl

```diff
@@ -7,8 +7,36 @@ behavior "remove_labels_on_reply" "remove_stale" {
     labels = ["waiting-reply"]
 }
 
+poll "stale_issue_closer" "waiting_reply_closer" {
+    schedule = "1 0 * * *"
```

Comment:
Same as other comments, prefer run once a day, leave the leftmost digit to `0`. Sorry about the confusion its related to the underlying library.